api/login/login.go

```diff
@@ -29,15 +31,18 @@ func createAccount(response http.ResponseWriter, request *http.Request) {
 	var acc Account
 	err := json.NewDecoder(request.Body).Decode(&acc)
 
+    // Check for JSON decoding errors or empty email.
 	if err != nil || acc.Email == "" {
 		return
 	}
 
+    // Execute a database query to insert the new account.
 	result, err := database.Execute("INSERT INTO Accounts (email, password) VALUES (?, SHA2(?, 256))", acc.Email, acc.Password)
 	if err != nil {
 		return
 	}
 
+	// Check if any rows were affected by the database operation.
```

Comment:
If you want to comment this it's probably more effective to say that no rows affected = account already exists (iirc)